### ISSUE TITLE
Status bar does not determine branch or remote

### ISSUE THREAD

**Issue by jeffbyrnes** [jisaacks/GitGutter#461]
## Description

The new status bar text seems unable to determine my current branch, nor what branch it tracks.

Here’s a sample output of the command it runs (per jisaacks/GitGutter@cbf9485aab):

```bash
♠ git status -b -s -u
## master...origin/master [ahead 1]
 M modules/api/data.tf
 M modules/api/main.tf
 M modules/api/variables.tf
 M modules/devsite/main.tf
?? global/dns_darksky_com.tf
?? modules/api/outputs.tf
?? modules/vpn/consul.tf
?? modules/vpn/main.tf
?? modules/vpn/outputs.tf
?? modules/vpn/variables.tf
?? terraform stuff.mdown
```

I should note that output is colorized. I.e., the `master` text is in one color, `origin/master` is in another, etc.

The status bar output reads:

> In terraform on unknown, tracking None, File is committed…

## Support Info

- Sublime Text 3143
- Platform: osx
- Arch: x64
- GitGutter 1.7.7
- Install via PC: True
- git version 2.14.2
- mdpopups 2.1.1
- markdown 2.6.9
- pygments 2.1a0
- jinja2 2.8

## Steps to Reproduce Issue

1. Install GitGutter & configure it to show the current branch and that branch’s remote
2. Open a file in a Git repository
3. Read the status bar text

**Comment by deathaxe** [comment]
I've passed your status lines to `parse_output()` function which converts it to the variables passed to jinja2. Printing all values returns: `master origin/master 1 None 7 0 4 0` - the expected behavior.

I added `{{remote}}` to my GitGutter.sublime-settings and committed it to my local branch to be one ahead of the remote as in your example. Status bar text renders well.

If you just updated, you might need to restart ST. The pattern used to parse the header line is compiled as global variable and therefore might not be loaded correctly.

**Comment by jeffbyrnes** [comment]
@deathaxe hmm. Just restarted Sublime, same thing. Attaching a screenshot.

![screen shot 2017-10-17 at 3 34 40 pm](https://user-images.githubusercontent.com/260553/31685372-b3736f90-b350-11e7-96f9-1a985b6a8e8f.png)

**Comment by deathaxe** [comment]
The reason was identified in #462 and is fixed in GitGutter 1.7.8 which was committed some minutes ago. Should arrive soon.

**Comment by jeffbyrnes** [comment]
Oh, great! Closing in favor of that issue.
